### ISSUE TITLE
Cannot connect to WebUI

### ISSUE THREAD

**Issue by williamgeurts** [hassio-addons/addon-unifi#184]
# Problem/Motivation

> (Why the issue was filed)

## Expected behavior

> (What you expected to happen)

## Actual behavior

> (What actually happened)

## Steps to reproduce

> (How can someone else make/see it happen)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by williamgeurts** [comment]
Hi,
I have the Unifi controller for a few month running. I don’t access the webpage often, only if I think that I experience a problem. Yesterday I wanted to add a new AP, but the webinterface was not reachable. I use the https with the self signed certificate. Everything I tried didn’t work (snapshot restore, restart). I deleted the controler and reinstalled it, then it works, but with an empty config. The logging shows a normal startup sequence ending in [23:28:10] INFO: Starting the UniFi Controller...
How can I find out what is wrong or how can I install a new Controller and use the old config.
